src/__internal__/form-field/form-field.component.tsx

```diff
@@ -160,15 +160,16 @@ const FormField = ({
     };
   }, [id, setError, setWarning, setInfo, error, warning, info]);
 
-  const fieldHelp = fieldHelpContent ? (
-    <FieldHelp
-      labelInline={inlineLabel}
-      labelWidth={labelWidth}
-      id={fieldHelpId}
-    >
-      {fieldHelpContent}
-    </FieldHelp>
-  ) : null;
+  const fieldHelp =
+    fieldHelpContent && !validationRedesignOptIn ? (
```

Comment:
@robinzigmond if you are able to do this with `Textbox`, then it would be a good idea to assess which props related to the old validation pattern remain in Carbon and will need removing in the future.

Reply:
`fieldHelp` doesn't render anything in the new style validation. It only works for old style. I agree with a separate task to identify the props related to the old style validation. It'll make decoupling it easier when it comes to deprecation time. 

Screenshot to demonstrate how `fieldHelp` doesn't render in new validations. 
![Screenshot 2023-06-22 at 14 29 56](https://github.com/Sage/carbon/assets/56251247/bc694e14-6ad3-473e-ba39-3fbd68506147)